File: generators/java/src/main/java/dqualizer/dqlang/archive/loadtesttranslator/dqlang/modeling/RuntimeQualityAnalysisDefintion.java
package dqualizer.dqlang.archive.loadtesttranslator.dqlang.modeling;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.ToString;

/**
 * Java class for the modeling
 */
@Getter
@ToString
public class RuntimeQualityAnalysisDefintion {

    private int version;
    private String context;
    private String environment;
    @JsonProperty("runtime_quality_analysis")
    private RuntimeQualityAnalysis rqa;
}

File: src/main/kotlin/io/github/dqualizer/dqlang/archive/loadtesttranslator/dqlang/modeling/RuntimeQualityAnalysisDefintion.kt
package io.github.dqualizer.dqlang.archive.loadtesttranslator.dqlang.modeling

/**
 * Java class for the modeling
 */
import com.fasterxml.jackson.annotation.JsonProperty
import lombok.Getter
import lombok.ToString

@Getter
@ToString
class RuntimeQualityAnalysisDefintion {
    val version = 0
    val context: String? = null
    val environment: String? = null

    @JsonProperty("runtime_quality_analysis")
    val rqa: io.github.dqualizer.dqlang.archive.loadtesttranslator.dqlang.modeling.RuntimeQualityAnalysis? = null
}
